c2corg_common/attributes.py

```diff
@@ -608,9 +608,9 @@
 ]
 
 quality_types = [
+    'empty',
```

Comment:
`empty` means what? Isn't this like leaving the field `null` and not using required?